src/core/components/SideDrawer/SideDrawer.native.js

```diff
@@ -0,0 +1 @@
+export default null;
```

Comment:
VxpSideDrawer.native.vue has duplicate implementation with VxpSideDrawer.vue. I think we can use RadSideDrawer in this file and delete VxpSideDrawer.native.vue file